content/conda.md

```diff
@@ -51,6 +51,16 @@ Anaconda is more convenient at the beginning since it provides most packages out
     - [Linux](https://docs.conda.io/projects/continuumio-conda/en/latest/user-guide/install/linux.html#updating-anaconda-or-miniconda)
 
 
+### If you have Anaconda and you are using it regularly
+
+In this case you can probably go through most of the workshop and most of the
+packages that we will need already are part of Anaconda. If some package is
+missing, you can then fetch it with the Anaconda Navigator. One disadvantage of
+using Anaconda is however, that you are more likely to install everything into
+the base environment whereas the Miniconda approach suggests to use more
+isolated environments without modifying your base environment.
```

Comment:
A person can still make the coderefinery environment if they want - I guess we can give that option?
```suggestion
You can use your existing Anaconda install and create the Coderefinery
environment as the {ref}`next page <conda-environment>` says.

Or, you can probably go through most of the workshop and most of the
packages that we will need already are part of Anaconda. If some package is
missing, you can then fetch it with the Anaconda Navigator. One disadvantage of
using Anaconda is however, that you are more likely to install everything into
the base environment whereas the Miniconda approach suggests to use more
isolated environments without modifying your base environment.
```